tests/mime/mime-stream-depth/README.md

```diff
@@ -0,0 +1,8 @@
+# Description
+
+Test mime base64 decoding completeness when stream depth is reached
```

Comment:
Is this on the standard stream depth setting for now? 64 mb?

While I see that we're not trying to store any files, would filestore's stream depth settings also affect this (as it should ideally override stream's depth settings)?

So, should we have a suricata.yaml in this test? Or would we never change the standard settings for depths? wdyt?

Reply:
Yes, it is for standard depth which is 1Mb

> would filestore's stream depth settings also affect this (as it should ideally override stream's depth settings)?

Maybe

> So, should we have a suricata.yaml in this test? 

Yes, or `--set stream.depth=1mb`

Reply:
`stream.reassembly.depth` in facts